.prow.yaml

```diff
@@ -7,7 +7,7 @@ presubmits:
       preset-goproxy: "true"
     spec:
       containers:
-        - image: ghcr.io/kcp-dev/infra/build:1.22.5-1
+        - image: ghcr.io/kcp-dev/infra/build:1.22.7-1
```

Comment:
This needs to be published first here:
https://github.com/kcp-dev/infra/pkgs/container/infra%2Fbuild before we can update

Reply:
https://github.com/kcp-dev/infra/pull/73 is supposed to do it